### DISCUSSION TITLE
HTTP 503 response during uploading large files

### DISCUSSION THREAD

**Discussion by everwisher** [abraunegg/onedrive#1994] [Q&A] [answered]
I have a bunch of very large files (over 50GB) to sync with/upload to one of my SharePoint sites. When I used `onedrive --monitor`, there are many errors during the process that interrupt uploading of certain files: `Error Message: HTTP request returned status code 503 (Service Unavailable)`. 

Any ideas to deal with this?

**Comment by abraunegg** [comment, marked as answer]
@everwisher 
> Any ideas to deal with this?

A 503 error is something generated by the Microsoft OneDrive service you are using .. it has 100% nothing to do with this client. You also need to review your Service Health Dashboard (SHD) to ensure that there is no Microsoft infrastructure outage.

As you are using SharePoint, thus at least a business account which has support, your *only* avenue here is to open a support request with Microsoft using your 'business' account support process. I cannot assist at all.

You will most likely need to run the client in 'debug' mode and provide to Microsoft this debug log so that they can fix their API / SharePoint system and debug what server in their environment is failing.

**Reply by everwisher** [reply]
Thanks I'll take a look at the SHD thing to see if I can figure out anything
